Add on-ramps page (#349)
LiskHQ/lisk-docs: docs/lisk-chain/lisk-tools/on-ramps.md

---
title: On-Ramps
description: Documentation for various On-Ramps for Lisk.
keywords:
  [
    On-Ramps,
    On-Ramp,
    Lisk,
    Buy LSK,
    Get LSK
  ]
---

# On-Ramps

On-Ramps (fiat-to-crypto) and Off-Ramps (crypto-to-fiat) are services that act as a gateway between traditional fiat currencies (like USD or EUR) and the world of digital assets.

> It allows users to convert their local money into cryptocurrencies and vice versa through familiar payment methods such as bank transfers, credit cards, or Apple Pay. 

Reviewer: ```suggestion
They allow users to convert their local money into cryptocurrencies and vice versa through familiar payment methods such as bank transfers, credit cards, or Apple Pay. 
```

> The following On- and Off-ramp service provider support the Lisk network already:

Reviewer: ```suggestion
The following On- and Off-ramp service providers support the Lisk network already:
```

## Banxa

[Banxa](https://banxa.com/) is a global on/off-ramp integrating LSK, USDT and USDC on the Lisk chain.

> Banxa provides easy seamless integration of crypto payment services into any application by offering well-documented APIs and SDKs.

Reviewer: ```suggestion
Banxa provides easy, seamless integration of crypto payment services into any application by offering well-documented APIs and SDKs.
```

## Moonpay

[MoonPay](https://www.moonpay.com/) is the leading ramps provider in the space and continues to be the most popular gateway for crypto and NFT applications across the world. With the MoonPay fiat-to-crypto On-Ramp, users can seamlessly purchase cryptocurrencies directly from your website or web3 application.

Moonpay offers a wide range of cryptocurrencies to buy and sell, see [List of supported cryptocurrencies](https://dev.moonpay.com/docs/list-of-supported-cryptocurrencies)

## Fonbnk

[Fonbnk](https://www.fonbnk.com/) provides on- and off-ramps for USDT across Africa.

Fonbnk pairs prepaid mobile payments with stablecoins to eliminate fraud, reduce FX costs, and deliver real-time, chargeback-free global settlements.

## Kotani Pay

[Kotani Pay](https://kotanipay.com/) provides a reliable on-ramp and off-ramp solution, allowing you to easily convert between cryptocurrencies and local fiat currencies across Africa.

## OnRamp Money

> [OnRamp Money](https://onramp.money/) is an emerging market focused on/off-ramp with many local payment methods; integrating LSK, USDT and USDC on the Lisk chain.

Reviewer: ```suggestion
[OnRamp Money](https://onramp.money/) is an emerging-market-focused on/off-ramp with many local payment methods; integrating LSK, USDT and USDC on the Lisk chain.
```

---
title: On-Ramps
description: Documentation for various On-Ramps for Lisk.
keywords:
  [
    On-Ramps,
    On-Ramp,
    Off-Ramps,
    Off-Ramp,
    Lisk,
    Buy LSK,
    Get LSK
  ]
---

# On-Ramps

On-Ramps (fiat-to-crypto) and Off-Ramps (crypto-to-fiat) are services that act as a gateway between traditional fiat currencies (like USD or EUR) and the world of digital assets.
They allow users to convert their local money into cryptocurrencies and vice versa through familiar payment methods such as bank transfers, credit cards, or Apple Pay. 

The following On- and Off-ramp service providers support the Lisk network already:

## Banxa

[Banxa](https://banxa.com/) is a global on/off-ramp integrating LSK, USDT and USDC on the Lisk Chain.
Banxa provides easy, seamless integration of crypto payment services into any application by offering well-documented APIs and SDKs.

> ## Moonpay

Reviewer: ```suggestion
## MoonPay
```

[MoonPay](https://www.moonpay.com/) is a leading ramps provider and integrates LSK on the Lisk Chain.
With the MoonPay fiat-to-crypto On-Ramp, users can seamlessly purchase cryptocurrencies directly from your website or web3 application.

MoonPay offers a wide range of cryptocurrencies to buy and sell, see [List of supported cryptocurrencies](https://dev.moonpay.com/docs/list-of-supported-cryptocurrencies)

## Fonbnk

[Fonbnk](https://www.fonbnk.com/) provides on- and off-ramps for USDT on the Lisk Chain across Africa.

Fonbnk pairs prepaid mobile payments with stablecoins to eliminate fraud, reduce FX costs, and deliver real-time, chargeback-free global settlements.

## Kotani Pay

[Kotani Pay](https://kotanipay.com/) is a payment platform that provides a fiat to stablecoin on-ramp and off-ramp to local payment channels in Africa.

> Kotani Pay allows blockchain protocols, dApps, and blockchain fintechs to easily integrate to payment solutions into their applications, websites, or web plugins.

Reviewer: ```suggestion
Kotani Pay allows blockchain protocols, dApps, and blockchain fintechs to easily integrate payment solutions into their applications, websites, or web plugins.
```

Kotani Pay integrates USDT and USDC on the Lisk Chain.

## OnRamp Money

> [OnRamp Money](https://onramp.money/) is an emerging-market-focused on/off-ramp with many local payment methods; integrating LSK, USDT and USDC on the Lisk Chain.

Reviewer: ```suggestion
[OnRamp Money](https://onramp.money/) is an emerging-market-focused on/off-ramp with many local payment methods. It integrates LSK, USDT and USDC on Lisk Chain.
```

## Noblocks

[Noblocks](https://www.noblocks.xyz/) is an intuitive interface that enables users to create payment orders for instant stablecoin-fiat exchanges.
Noblocks integrates USDT on the Lisk Chain.
Built on Paycrest's decentralized liquidity network, it supports mobile money and bank settlements across multiple regions, including Kenya (KES), Uganda (UGX), Nigeria (NGN), Tanzania (TZS), and more.

## Aliniex

[Alix Pay](https://alixpay.com/), a core service of Aliniex, is designed to simplify crypto on and off-ramp solutions for users worldwide. 
Alix Pay integrates LSK on the Lisk Chain.

> Since its inception in 2017, Aliniex has established itself as a trusted cryptocurrency exchange, with a particularly strong presence in Southeast Asia. 

Reviewer: ```suggestion
Since its inception in 2017, Aliniex has established itself as a trusted cryptocurrency exchange with a particularly strong presence in Southeast Asia. 
```

---
title: On-Ramps
description: Documentation for various On-Ramps for Lisk.
keywords:
  [
    On-Ramps,
    On-Ramp,
    Off-Ramps,
    Off-Ramp,
    Lisk,
    Buy LSK,
    Get LSK
  ]
---

# On-Ramps

On-Ramps (fiat-to-crypto) and Off-Ramps (crypto-to-fiat) are services that act as a gateway between traditional fiat currencies (like USD or EUR) and the world of digital assets.
They allow users to convert their local money into cryptocurrencies and vice versa through familiar payment methods such as bank transfers, credit cards, or Apple Pay. 

The following On- and Off-ramp service providers support the Lisk network already:

## Banxa

> [Banxa](https://banxa.com/) is a global on/off-ramp integrating LSK, USDT and USDC on the Lisk chain.

Reviewer: "Lisk Chain" is a proper noun and should be capitalized and always written without article, see https://lisk.com/chain/

There are several occurrences of this issue here.

Banxa provides easy, seamless integration of crypto payment services into any application by offering well-documented APIs and SDKs.

## Moonpay

[MoonPay](https://www.moonpay.com/) is a leading ramps provider and integrates LSK on the Lisk chain.
With the MoonPay fiat-to-crypto On-Ramp, users can seamlessly purchase cryptocurrencies directly from your website or web3 application.

> Moonpay offers a wide range of cryptocurrencies to buy and sell, see [List of supported cryptocurrencies](https://dev.moonpay.com/docs/list-of-supported-cryptocurrencies)

Reviewer: ```suggestion
MoonPay offers a wide range of cryptocurrencies to buy and sell, see [List of supported cryptocurrencies](https://dev.moonpay.com/docs/list-of-supported-cryptocurrencies)
```

## Fonbnk

[Fonbnk](https://www.fonbnk.com/) provides on- and off-ramps for USDT on the Lisk chain across Africa.

Fonbnk pairs prepaid mobile payments with stablecoins to eliminate fraud, reduce FX costs, and deliver real-time, chargeback-free global settlements.

## Kotani Pay

> [Kotani Pay](https://kotanipay.com/) provides a reliable on-ramp and off-ramp solution, allowing you to easily convert between cryptocurrencies and local fiat currencies across Africa.

Reviewer: Are we using "you" and this style of informal language generally in the documentation?

Note that these are founder / dev docs, so "allowing you to easily convert between cryptocurrencies" doesn't make sense in my opinion.

Author: Updated the description to be more targeted to founders, please check!

> Integrates USDT and USDC on the Lisk chain.

Reviewer: A subject is missing here.

## OnRamp Money

[OnRamp Money](https://onramp.money/) is an emerging-market-focused on/off-ramp with many local payment methods; integrating LSK, USDT and USDC on the Lisk chain.

## Noblocks

[Noblocks](https://www.noblocks.xyz/) is an intuitive interface that enables users to create payment orders for instant stablecoin-fiat exchanges.
Integrates USDT on the Lisk chain.
Built on Paycrest's decentralized liquidity network, it supports mobile money and bank settlements across multiple regions, including Kenya (KES), Uganda (UGX), Nigeria (NGN), Tanzania (TZS), and more.

## Aliniex

[Alix Pay](https://alixpay.com/), a core service of Aliniex, is designed to simplify crypto on and off-ramp solutions for users worldwide. 

> Integrates LSK on the Lisk chain.

Reviewer: A subject is missing.
